microk8s-resources/actions/enable.gpu.sh

```diff
@@ -20,8 +41,10 @@ echo "Installing NVIDIA Operator"
 "$SNAP/microk8s-helm3.wrapper" repo add nvidia https://nvidia.github.io/gpu-operator
 "$SNAP/microk8s-helm3.wrapper" repo update
 "$SNAP/microk8s-helm3.wrapper" install gpu-operator nvidia/gpu-operator \
+  --version=v1.7.0 \
```

Comment:
It will be good if the operator/toolkit versions are controlled through arguments too, so we don't need to change the script with each release of operator.

Reply:
Hey @shivamerla, we'll look at adding this as an argument in the future.  Obviously, we need to hard code a default for a testable / supportable release build.